com.nest/camera.js

```diff
@@ -59,6 +59,10 @@ const CameraDevice = class NestCameraDevice extends Tp.BaseDevice {
         return stream;
     }
 
+    get_state() {
+        throw new Error ('Sorry! Your device doesn\'t support querying the state.');
```

Comment:
Do you plan to implement this? After all, we can change the state, so we should be able to query it.

Reply:
Yup sure, where can I find the nest camera api?

Reply:
https://developers.nest.com/reference/api-camera

Reply:
I'm reinstating the old Nest security_camera dataset and manifest for now, since HA doesn't have a camera device.

For now, the Nest security_camera doesn't extend the generic motion sensor yet. Because the Nest security_camera has a `stream  := monitor ((@com.nest.security_camera.current_event()), has_motion == true)` command that sort of overlaps with motion sensor commands.

Reply:
The overlap is the whole point: we need to reannotate potentially conflicting sentences, and we need to decide how to do it.